ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -162,6 +164,7 @@ public static ProtocolSpecBuilder frontierDefinition(
         .blockImporterBuilder(MainnetBlockImporter::new)
         .blockHeaderFunctions(new MainnetBlockHeaderFunctions())
         .miningBeneficiaryCalculator(BlockHeader::getCoinbase)
+        .jumpDestCacheConfig(jumpDestCacheConfiguration)
```

Comment:
/Config/Configuration/  

I know both are used in the codebase, but would be good if we could start being consistent at least within the same feature. p.s don't mind which!